schema/me/consignments/asset.js

```diff
@@ -0,0 +1,24 @@
+import { GraphQLObjectType, GraphQLString, GraphQLNonNull } from "graphql"
+
+export const AssetType = new GraphQLObjectType({
+  name: "Asset",
+  description: "An asset which is assigned to a consignment submission",
+  fields: {
+    id: {
+      description: "Convection id.",
```

Comment:
Maybe `"Convection asset ID"` to be consistent with the rest of the descriptions here?

Reply:
Also, we may want the `asset_type` here as well.